<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWin</class>
 <widget class="QDialog" name="PlotWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Data</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>
    <normalon>icon.png</normalon>icon.png</iconset>
  </property>
  <widget class="QCustomPlot" name="plot" native="true">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>451</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Arduino Plotter</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>421</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="plotButton">
      <property name="text">
       <string>Plot</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLCDNumber" name="valueLcdNumber">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="gridCheckBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>220</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Grid Off</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCurrentValue">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Current Value</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="recordCheckBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Record CSV Data</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
